packages/core/src/shared/sam/utils.ts

```diff
@@ -108,6 +108,35 @@ export async function updateRecentResponse(
         getLogger().warn(`sam: unable to save response at key "${key}": %s`, err)
     }
 }
+
+const buildProcessMementoRootKey = 'samcli.build.processes'
```

Comment:
Let's change this to something like

const BUILD_PROCESS_MEMENTO_ROOT_KEY = 'samcli.build.processes'; and maybe put it in the samutils if it makes sense to. We can also do the same with the 'global' as we should limit the amount of constant strings we have

Reply:
The above format wouldn't work since eslint only allows camelCase or PascalCase formats and the constant is defined in the sam/utils file. Are you referring to a different utils file?

Reply:
Ideally, we'd put this in a constants.ts file, but if not we can disable eslint for this line. The convention is that constants should be in all caps.